docs/jplspec/jplspec.rst

```diff
@@ -0,0 +1,311 @@
+   >>> print(response.info)
+      <Table length=7>
+      name  dtype    unit
+    ----- ------- -------
+     FREQ float64     MHz
+      ERR float64     MHz
+    LGINT float64 MHz nm2
+       DR   int64
+      ELO float64  1 / cm
+      GUP   int64
+      TAG   int64
+    QNFMT   int64
+      QN'   int64
+      QN"   int64
+
+These come in handy for converting to other units easily, an example using a
+simplified version of the data above is shown below:
+
+.. code-block:: python
+
+   >>> print (response)
+       FREQ      ERR     ELO
+       MHz       MHz    1 / cm
+    ----------- ------- -------
+    115271.2018  0.0005     0.0
+    345795.9899  0.0005  11.535
+    461040.7682  0.0005 23.0695
+   >>> response['FREQ'].quantity
+   <Quantity [115271.2018,345795.9899,461040.7682] MHz>
+   >>> response['FREQ'].to('GHz')
+   <Quantity [115.2712018,345.7959899,461.0407682] GHz>
+
+The parameters and response keys are described in detail under the
+Reference/API section.
+
+Looking Up More Information from the catdir.cat file
+------------------------------------------------------
+
+If you have found a molecule you are interested in, the TAG field
+in the results provides enough information to access specific
+molecule information such as the partition functions at different
+temperatures. Keep in mind that a negative TAG value signifies that
+the line frequency has been measured in the laboratory
+
+.. code-block:: python
+
+   >>> import matplotlib.pyplot as plt
+   >>> from astroquery.jplspec import JPLSpec
+   >>> result = JPLSpec.get_species_table()
+   >>> mol = result[result['TAG'] == 28001] #do not include signs of TAG for this
+   >>> print(mol)
+   TAG   NAME NLINE  QLOG1   QLOG2  ...  QLOG5   QLOG6   QLOG7  VER
+   int64 str13 int64 float64 float64 ... float64 float64 float64 str2
+   ----- ----- ----- ------- ------- ... ------- ------- ------- ----
+   28001    CO    91  2.0369  1.9123 ...  1.1429  0.8526  0.5733   4*
+
+You can also access the temperature of the partition function
+through metadata:
+
+.. code-block:: python
+
+   >>> result['QLOG2'].meta
+   {'Temperature (K)' : 225}
+   >>> result.meta
+   {'Temperature (K)': [300, 225, 150, 75, 37.5, 18.5,
+                        9.375]}
+
+One of the advantages of using JPLSpec is the availability in the catalog
+of the partition function at different temperatures for the molecules. As a
+continuation of the example above, an example that accesses and plots the
+partition function against the temperatures found in the metadata is shown
+below:
+
+.. code-block:: python
+
+   >>> temp = result.meta['Temperature (K)']
+   >>> part = list(mol['QLOG1','QLOG2','QLOG3', 'QLOG4', 'QLOG5','QLOG6',
+                       'QLOG7'][0])
+   >>> plt.scatter(temp,part)
+   >>> plt.xlabel('Temperature (K)')
+   >>> plt.ylabel('Partition Function Value')
+   >>> plt.title('Parititon Fn vs Temp')
+   >>> plt.show()
+
+
+.. figure:: images/docplot_jplspec.png
```

Comment:
actually it would be nice to let the docs generate the figure. However I suggest to do that in a follow-up PR rather than holding this up from merging.

Reply:
No problem, I can definitely work on this for a follow up PR as you mentioned.